### DISCUSSION TITLE
2022/10/22/MATH430/

### DISCUSSION THREAD

**Discussion by giscus** [Jacob-SouthernCity/Jacob-SouthernCity.github.io#2] [Announcements]
# 2022/10/22/MATH430/

Description Intructor: Masoud Zargar Course description: This is a course on (elementary) number theory, covering the basic problems with which this subject deals, as well as how it is related to othe

https://jacob-southerncity.github.io/2022/10/22/MATH430/

**Comment by Jacob-SouthernCity** [comment]
Currently the notes is still a in process stuff, so you might need to check this site on a weekly basis if you need the up to date notes.

**Comment by Awinfredwa** [comment]
The note is so useful! Thanks a lot!

**Comment by huihuibuhui1** [comment]
The note is so useful! Thanks a lot!

**Reply by Jacob-SouthernCity** [reply]
模版是吧
